src/main/webapp/resources/js/pages/launch/LaunchDetails.jsx

```diff
@@ -28,6 +28,9 @@ export function LaunchDetails() {
       <Form.Item label={i18n("LaunchDetails.description")} name="description">
         <Input.TextArea />
       </Form.Item>
+      <Form.Item name="emailPipelineResult" valuePropName="checked">
```

Comment:
As mentioned above, the email pipeline result is now a select with 3 options so this will need to be updated

Reply:
I will update this in a separate PR